documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -31,23 +51,35 @@ kind: Secret
 metadata:
   name: <secret>
   namespace: {namespace}
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
   labels:
-    createdForProviderType: <provider_type> <1>
+    createdForProviderType: <provider_type> <2>
 type: Opaque
 stringData:
-  user: <user> <2>
-  password: <password> <3>
-  cacert: | <4>
+  user: <user> <3>
+  password: <password> <4>
+  domainName: <domain_name> <5>
+  projectName: <project_name> <6>
+  regionName: <region name> <7>
+  cacert: | <8>
     <ca_certificate>
-  url: <api_end_point>
-  thumbprint: <vcenter_fingerprint> <5>
-EOF
+  url: <api_end_point> <9>
+  thumbprint: <vcenter_fingerprint> <10>
 ----
-<1>  Specify the type of source provider. Allowed values are `ovirt` and `vsphere`. This label is needed to verify that the credentials are correct when the remote system is accessible and, for {rhv-short}, to retrieve the {manager} CA certificate when a third-party certificate is specified for `cacert`.
-<2> Specify the vCenter user or the {rhv-short} {manager} user.
-<3> Specify the user password.
-<4> {rhv-short} only: Specify the {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificater. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
-<5> VMware only: Specify the vCenter SHA-1 fingerprint.
+<1> The `ownerReferences` section is optional.
+<2>  Specify the type of source provider. Allowed values are `ovirt`, `vsphere`, and `openstack`. This label is needed to verify the credentials are correct when the remote system is accessible and, for {rhv-short}, to retrieve the {manager} CA certificate when a third-party certificate is specified.
+<3> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user.
+<4> Specify the user password.
+<5> {osp} only: Specify the domain name.
+<6> {osp} only: Specify the project name.
+<7> {osp} only: Specify the name of the {osp} region.
+<8> {rhv-short} and {osp} only: {manager} CA certificate unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
```

Comment:
The description about CA certification here is only for RHV. How about saying like this "For {rhv-short}, it's {manager} CA certificate unless....  For {osp}, it's the CA certificate for connecting to the source environment."